src/_guides/language/language-tour.md

```diff
@@ -1757,15 +1766,17 @@ might be returned.
 
 The previous example is equivalent to:
 
+<?code-excerpt "test/language_tour/browser_test.dart (cascade-operator-example-expanded)"?>
 {% prettify dart %}
-var button = querySelector('#button');
+var button = querySelector('button');
```

Comment:
same question

Reply:
same answer :)

Reply:
I'll switch to `#confirm` instead of `button` as selector here too.